packages/crawler/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "crawler",
+  "version": "0.0.0",
+  "private": true,
+  "license": "UNLICENSED",
+  "scope": "@porsche-design-system",
+  "author": "Dr. Ing. h.c. F. Porsche AG",
+  "scripts": {
+    "start": "ts-node src/index.ts",
+    "test:e2e:puppeteer": "jest --colors -c tests/e2e/config/jest.config.ts",
+    "test:e2e": "concurrently -k -s 'first' -n 'cdn,puppeteer' 'serve-cdn' 'yarn test:e2e:puppeteer'",
+    "clean:node_modules": "rm -rf ./node_modules && rm -rf ./yarn-error.log"
```

Comment:
`clean:node_modules` is a child command, so there should be a parent command called `clean`.

Reply:
Fixed